### PR TITLE
[tests-only] added e2e test to receive two shares with same name

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
In this PR,
- removed web UI test to accept two shares with same name
- added e2e test to accept two shares with same name

## Related Issue

- https://github.com/owncloud/web/issues/10259

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

### CODE REVIEW

File: tests/e2e/cucumber/features/smoke/shares/share.feature [hunk 1]
```diff
@@ -158,3 +158,44 @@ Feature: share
...
+      | test-folder  | Brian     | user | Can view |
+    And "Carol" logs out
+    When "Brian" navigates to the shared with me page
+    Then following resources should be displayed in the Shares for user "Brian"
+      | resource         |
+      | testfile.txt     |
+      | testfile (1).txt |
+    # https://github.com/owncloud/ocis/issues/8471
+    #  | test-folder      |
+    #  | test-folder (1)  |
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/smoke/shares/share.feature:200]: issue is only for folder shares?

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md [hunk 2]
```diff
@@ -24,7 +24,3 @@ Other free text and markdown formatting can be used elsewhere in the document if
 ### [empty subfolder inside a folder to be uploaded is not created on the server](https://github.com/owncloud/web/issues/6348)
 
 - [webUIUpload/upload.feature:43](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/upload.feature#L43)
-
-### [PROPFIND to sub-folder of a shared resources with same name gives 404](https://github.com/owncloud/ocis/issues/3859)
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:28]: what is the status of this issue? I see it is still open

File: tests/e2e/cucumber/features/smoke/shares/share.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/e2e/cucumber/features/smoke/shares/share.feature:200]: I see that works
image

Comment (PrajwolAmatya) [reply tests/e2e/cucumber/features/smoke/shares/share.feature:200]: The issue might occur for files as well as in this issue https://github.com/owncloud/ocis/issues/8471.

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md (hunk 2 above)
Comment (PrajwolAmatya) [reply tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:28]: The mentioned issue doesn't exist on the latest oCIS master. The expected behavior `(1)` to be added on the second share with same name, and resources can be accessed via propfind. So, that issue is closed now.

File: tests/e2e/cucumber/features/smoke/shares/share.feature (hunk 1 above)
Comment (saw-jan) [reply tests/e2e/cucumber/features/smoke/shares/share.feature:200]: then will the test get flaky if we keep `testfile (1).txt`?

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md (hunk 2 above)
Comment (saw-jan) [reply tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:28]: Then please, find the API test coverage for that one and if there is none we have to add one

File: tests/e2e/cucumber/features/smoke/shares/share.feature (hunk 1 above)
Comment (PrajwolAmatya) [reply tests/e2e/cucumber/features/smoke/shares/share.feature:200]: In this scenario, the shares list for the user is:
```
testfile.txt
testfile (1).txt
test-folder
test-folder (2)
```

While i did it manually, the shares list for the user is:
```
testfile.txt
testfile (3).txt
test-folder
test-folder (1)
```
Not so sure why we get different results while testing manually and in automation.

**Review by saw-jan — APPROVED** [review]
LGTM

**Review by ScharfViktor — APPROVED** [review]

**Pull request merged**